sssoja/tv-bland
# moves intro out of core folder

### What changes have you made?
- Moved `Intro` out of `core` folder as this is not used throughout the site 
- Updated the pathname of the `Intro` import

### Which issue(s) does this PR fix?
Fixes #50 

### Screenshots (if there are design changes)
No design changes

### How to test
- Check that the `Intro` component is now in the `components` folder 
- When you open `localhost:3000` the updated pathname should not throw any errors
